assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -107,6 +107,9 @@ spec:
           --requestheader-group-headers=X-Remote-Group \
           --requestheader-username-headers=X-Remote-User \
           {{ end -}}
+          {{- range .Values.apiserver.extraFlags -}}
+          {{ . }} \
```

Comment:
Why use `\` at the end here but not in the other case?

Reply:
Because other case is an YAML list and this one is shell script. Also last parameter don't need `\`.

Reply:
Ohh, right! Sorry! :-)